go/libkb/json.go

```diff
@@ -80,6 +80,17 @@ func (f *JSONFile) Load(warnOnNotFound bool) error {
 		return err
 	}
 	f.jw = jsonw.NewWrapper(obj)
+
+	if runtime.GOOS == "android" {
```

Comment:
should we just do this on all platforms?

Reply:
I could go either way...I'm not aware of any issues on other platforms.

Only downside is cluttering up the logs, and maybe some privacy concerns about logging the information.  The device and username stuff is public, but maybe the fact that user X is also user Y?  or that user X uses proxy Z?

Reply:
good point.... ok android only...